phaseshifts/atorb.py

```
@@ -656,9 +639,7 @@ def update_config(self, conf):
         elif isinstance(conf, dict):
             self.__dict__.update(conf)
         else:
-            raise ValueError(
-                "conf argument '{}' is neither a " "str or dict instance".format(conf)
-            )
+            raise ValueError("conf argument '{}' is neither a " "str or dict instance".format(conf))
```

Ruff (0.14.10)

Prefer `TypeError` exception for invalid type

(TRY004)